src/starkware/committee/committee/committee/committee.py

```diff
@@ -393,6 +398,13 @@ async def main():
         hash_func=pedersen_hash_func,
         availability_gateway=availability_gateway,
     )
+
+    metrics_config = config.get("METRICS", {})
+    if metrics_config.get("enabled", False):
+        port = 8000
```

Comment:
If using the config from file, add all config to the file...
listen IP, listen port
also don't think the "enabled" key is mandatory, you can just check if the METRIC key exists, you can try using marshmallow serialization lib to handle the serialization and deserialization